sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/registry_support_classes.py

```diff
@@ -5,13 +5,25 @@
 from typing import List, Union
 
 from azure.ai.ml._restclient.v2022_10_01_preview.models import AcrDetails as RestAcrDetails
+from azure.ai.ml._restclient.v2022_10_01_preview.models import ArmResourceId as RestArmResourceId
 from azure.ai.ml._restclient.v2022_10_01_preview.models import RegistryRegionArmDetails as RestRegistryRegionArmDetails
 from azure.ai.ml._restclient.v2022_10_01_preview.models import StorageAccountDetails as RestStorageAccountDetails
+from azure.ai.ml._restclient.v2022_10_01_preview.models import StorageAccountType as RestStorageAccountType
+from azure.ai.ml._restclient.v2022_10_01_preview.models import SystemCreatedAcrAccount as RestSystemCreatedAcrAccount
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    SystemCreatedStorageAccount as RestSystemCreatedStorageAccount,
+)
+from azure.ai.ml._restclient.v2022_10_01_preview.models import UserCreatedAcrAccount as RestUserCreatedAcrAccount
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    UserCreatedStorageAccount as RestUserCreatedStorageAccount,
+)
 from azure.ai.ml.constants._registry import StorageAccountType
+from azure.ai.ml._utils._experimental import experimental
 
 
 # This exists despite not being used by the schema validator because this entire
 # class is an output only value from the API.
+@experimental
 class SystemCreatedAcrAccount:
```

Comment:
this docstring     :type arm_resource_id: str should mention None as default value

Reply:
Updated docstring with default value.